tea.py

```diff
@@ -0,0 +1,443 @@
+#
+# TEA class
+# 
+
+import pybamm
+import matplotlib.pyplot as plt
+#import numpy as np
+import matplotlib.patches as patches
+import pandas as pd
+
+
+class TEA:
+    """
+    A Techno-Economic Analysis class for estimation of cell metrics:
+
+    Parameters
+    ----------
+    parameter_values : dict
+        A dictionary of parameters and their corresponding numerical values.
+        Default is NoneParameters
+    """
+
+    def __init__(self, parameter_values):
+        self.parameter_values = parameter_values
+
+    def print_stack_breakdown(self, print_dict=False):
```

Comment:
why would you want to be able to say print is false in a function that is there to print something? I see this is for the dict. I'd just remove this.

I would just make another property `stack_breakdown_dataframe` that makes the dataframe if it doesn't exist already, and then returns it. users can then do what they like with it